app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -1153,6 +1161,16 @@ private void dispatchHomeScreen() {
         }
     }
 
+    /**
+     * Helper method to leave CommCare unconditionally
+     * @param context
```

Comment:
Please remove javadoc method argument annotations that don't contain any content